### PR TITLE
Update auto-generated documentation for 5.2.0-m1

### PR DESCRIPTION
**Pull request by sbesson**
This commit was generated by running

```
$ ant -f components/autogen/ gen-format-pages gen-structure-table
```

from the HEAD of develop. As the intermediate schema has no generated
documentation, the metadata pages are not regenerated.

### CODE REVIEW

File: docs/sphinx/formats/micro-manager.txt [hunk 1]
```diff
@@ -49,6 +49,7 @@ Utility: |Good|
 
 **Additional Information**
```
Comment (rleigh-dundee) [inline docs/sphinx/formats/micro-manager.txt]: Extra newline before for consistency with earlier section headings?

**Comment by rleigh-dundee** [comment]
All looks fine and good to merge.  One picky comment, it really makes little difference if it's fixed or not.

File: docs/sphinx/formats/micro-manager.txt (hunk 1 above)
Comment (sbesson) [inline docs/sphinx/formats/micro-manager.txt]: I think this file who was hand-edited. This extra line was generated by the autogen components from the metadata in `format-pages.txt`. It should be fine as it is.

**Comment by sbesson** [comment]
Thanks @rleigh-dundee, merging.

**Pull request merged**